commands/dap.go

```diff
@@ -0,0 +1,134 @@
+package commands
+
+import (
+	"context"
+	"io"
+	"net"
+	"os"
+
+	"github.com/containerd/console"
+	"github.com/docker/buildx/dap"
+	"github.com/docker/buildx/util/cobrautil"
+	"github.com/docker/buildx/util/ioset"
+	"github.com/docker/buildx/util/progress"
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func dapCmd(dockerCli command.Cli, rootOpts *rootOptions) *cobra.Command {
+	var options dapOptions
+	cmd := &cobra.Command{
+		Use:   "dap",
+		Short: "Start debug adapter",
+	}
+	cobrautil.MarkCommandExperimental(cmd)
+
+	flags := cmd.Flags()
+	flags.StringVar(&options.OnFlag, "on", "error", "When to pause the adapter ([always, error])")
```

Comment:
Isn't this `ExceptionBreakMode` sent via DAP protocol itself?

Reply:
Not currently implemented but yes ideally we would remove this and use `ExceptionBreakMode` to set the behavior.